interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -304,6 +307,13 @@ private void runTest(TestCases testCase) throws Exception {
         break;
       }
 
+      case GOOGLE_DEFAULT_CREDENTIALS: {
+        assertNotNull(customCredentialsType);
```

Comment:
Hmm... It doesn't seem right to require the user to specify customCredentialsType. Although I do see now that the test causes trouble when creating the channel.

Let's move the test from AbstractInteropTest to Tester in this file, and just have it manually create the Channel for itself. You'll need to shutdown the channel though, and that should probably be in a try-finally:

```
ManagedChannel channel = GoogleDefaultChannelBuilder.forAddress(serverHost, serverPort).build();
try {
  // do test
} finally {
  channel.shutdownNow();
}
```

Reply:
done